chore: update dependencies

tenstad opened the pull request.

tenstad committed:
chore: remove sample group with mytype resource (#15)

* chore: remove sample group with mytype resource

* chore: update prepare script

* chore: go mod tidy

renovate[bot] committed:
chore(deps): update gcr.io/distroless/static docker digest to f4a57e8 (#1)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump google.golang.org/protobuf from 1.31.0 to 1.33.0 (#4)

Bumps google.golang.org/protobuf from 1.31.0 to 1.33.0.

---
updated-dependencies:
- dependency-name: google.golang.org/protobuf
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update golang docker tag to v1.23 (#7)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update alpine docker tag to v3.20 (#5)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update actions/cache action to v4 (#21)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update fkirc/skip-duplicate-actions action to v5.3.1 (#6)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update actions/checkout action to v4 (#22)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update actions/setup-go action to v5 (#26)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update actions/github-script action to v7 (#25)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update codecov/codecov-action action to v5 (#28)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker/login-action action to v3

renovate[bot] committed:
chore(deps): update dependency ubuntu to v24 (#29)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker/setup-buildx-action action to v3 (#32)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker/setup-qemu-action action to v3 (#33)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update actions/upload-artifact action to v4 (#34)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update github/codeql-action action to v3 (#35)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update golangci/golangci-lint-action action to v6 (#36)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update korthout/backport-action action to v3 (#37)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update xt0rted/slash-command-action action to v2 (#38)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

erikgb committed:
ci: bump golang-ci-lint version (#40)

erikgb committed:
fix: bump Go version (#39)

erikgb committed:
fix: bump various dependencies (#31)

renovate[bot] committed:
chore(deps): update gcr.io/distroless/static docker digest to 5c7e2b4 (#43)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update alpine docker tag to v3.21 (#52)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Sjur <mariatsji@users.noreply.github.com>

erikgb committed:
fix: bump outdated Go dependencies (#69)

renovate[bot] committed:
fix(deps): update module sigs.k8s.io/controller-tools to v0.17.0 (#70)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Erik Godding Boye <egboye@gmail.com>

erikgb committed:
chore: bump submodule to latest commit (#73)

erikgb committed:
ci: migrate deprecated golangci-lint config (#76)

erikgb committed:
chore: bump transitive dependencies (#77)

renovate[bot] committed:
fix(deps): update module sigs.k8s.io/controller-runtime to v0.19.4 (#80)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Sjur Millidahl <sjur.millidahl@statnett.no>
Co-authored-by: Sjur <mariatsji@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update gcr.io/distroless/static docker digest to 3f2b64e (#98)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update module sigs.k8s.io/controller-tools to v0.17.1 (#99)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Erik Godding Boye <egboye@gmail.com>

renovate[bot] committed:
chore(deps): update module github.com/golangci/golangci-lint to v1.63.4 (#108)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update kubernetes packages to v0.32.1 (#118)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update module sigs.k8s.io/controller-runtime to v0.20.0 (#120)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update module sigs.k8s.io/controller-runtime to v0.20.1 (#125)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update module sigs.k8s.io/controller-tools to v0.17.2 (#140)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Erik Godding Boye <egboye@gmail.com>

renovate[bot] committed:
fix(deps): update module github.com/crossplane/crossplane-runtime to v1.19.0 (#141)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update module github.com/golangci/golangci-lint to v1.64.4 (#142)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update golang docker tag to v1.24 (#143)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update kubernetes packages to v0.32.2 (#145)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update module github.com/golangci/golangci-lint to v1.64.5 (#148)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update module sigs.k8s.io/controller-runtime to v0.20.2 (#149)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

tenstad committed:
ci: fix linting (#151)

renovate[bot] committed:
fix(deps): update module github.com/google/go-cmp to v0.7.0 (#154)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

tenstad committed:
chore: update controller template (#156)

renovate[bot] committed:
chore(deps): update module github.com/golangci/golangci-lint to v1.64.6 (#157)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update gcr.io/distroless/static docker digest to 95ea148 (#163)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update module sigs.k8s.io/controller-runtime to v0.20.3 (#159)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update module github.com/golangci/golangci-lint to v1.64.8 (#160)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update kubernetes packages to v0.32.3 (#161)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump golang.org/x/net from 0.34.0 to 0.36.0 (#162)

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

erikgb committed:
chore: upgrade and fix golangci-lint (#169)

renovate[bot] committed:
fix(deps): update module sigs.k8s.io/controller-runtime to v0.20.4 (#168)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update gcr.io/distroless/static docker digest to 3d0f463 (#170)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update module sigs.k8s.io/controller-tools to v0.17.3 (#171)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Sjur <mariatsji@users.noreply.github.com>
Co-authored-by: Erik Godding Boye <egboye@gmail.com>

renovate[bot] committed:
chore(deps): update module github.com/golangci/golangci-lint to v2.1.1 (#172)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

tenstad committed:
Revert "chore: remove sample group with mytype resource (#15)"

This reverts commit 563b2779a3c6c029980ed880e27d142e1cbff5ee.

tenstad commented on .github/workflows/ci.yml:
```diff
@@ -73,23 +72,24 @@ jobs:
         with:
-          version: ${{ env.GOLANGCI_VERSION }}
+          version: v2.1.1
+          args: --timeout 5m
```
We could revert back to the `${{ env.GOLANGCI_VERSION }}` here, as I removed the renovate marker. And maybe also not pin the version. Depending on how much we want to differ the upstream.

erikgb replied:
Yes, let's do it!

erikgb replied:
Unpinning also works for me. Even simpler. I am pretty sure the action runs the "latest" version by default.

erikgb reviewed (approved):
LGTM

tenstad committed:
chore: revert to env var for go ci version

tenstad commented:
Squashed as the whole thing got a bit messy, and to avoid creating a lot of references from commit messages to the upstream project, as just pushing all of them accidentally referenced a lot of PRs/issues 😅 The references were luckily removed once i deleted my fork 🙏
